resources/[gameplay]/gcshop/skin/skins_client.lua

```diff
@@ -102,7 +102,7 @@ function onShopInit ( tabPanel )
 		end
 	end	
 	guiCreateLabel(0.4274,0.0505,0.45,0.05,"Skin price: 3000 GCs",true,skinTab)
-	guiCreateLabel(0.4274,0.1009,0.45,0.05,"You can have as many skins as you want",true, skinTab)
+	guiCreateLabel(0.4274,0.1009,0.45,0.05,"You can have as many skins as you want, but you will only be able to use 1 skin in a time.",true, skinTab)
```

Comment:
Change to ", but you are only able to use 1 skin at a time.". Better English (correct me if I'm wrong)